### PR TITLE
Change memory requests & limits to units of 0.1G

### PR DESCRIPTION
**Pull request by fabubaker**
This is to satisfy the Azure ACI limitation of memory requests having granularity of 0.1G.

### CODE REVIEW

File: helm/rayyan/values.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ workers:
     resources:
       requests:
         cpu: 400m
-        memory: 2.5Gi
+        memory: 2.6G
```
[the remark this answers is not in the document]
Comment (hammady) [reply helm/rayyan/values.yaml]: Indeed 2.5Gi should still be valid.

File: helm/rayyan/values.yaml [hunk 2]
```diff
@@ -55,7 +55,7 @@ workers:
     resources:
       requests:
         cpu: 200m
-        memory: 512Mi
+        memory: 0.6G
```
Comment (hammady) [inline helm/rayyan/values.yaml]: 0.5Gi

File: helm/rayyan/values.yaml [hunk 3]
```diff
@@ -133,10 +133,10 @@ servers:
     resources:
       limits:
         cpu: 100m
-        memory: 128Mi
+        memory: 0.2G
```
Comment (hammady) [inline helm/rayyan/values.yaml]: Is the increment 0.1Gi or 0.1G? So powers of 2 or 10?

Comment (fabubaker) [reply helm/rayyan/values.yaml]: 0.1G, not Gi.

File: helm/rayyan/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (fabubaker) [reply helm/rayyan/values.yaml]: `2.5 Gi = 2.68435 G`. If anything I think it needs to be `2.7 G`.

File: helm/rayyan/values.yaml [hunk 4]
```diff
@@ -36,7 +36,7 @@ workers:
     resources:
       requests:
         cpu: 200m
-        memory: 512Mi
+        memory: 0.6G
```
[the remark this answers is not in the document]
Comment (fabubaker) [reply helm/rayyan/values.yaml]: ACI needs it to be in increments of `0.1G`, so `512 Mi = 536.871 M ≈ 600 M = 0.6 G`.

File: helm/rayyan/values.yaml (hunk 2 above)
Comment (fabubaker) [reply helm/rayyan/values.yaml]: ACI needs it to be in increments of `0.1G`, so `512 Mi = 536.871 M ≈ 600 M = 0.6 G`.

File: helm/rayyan/values.yaml (hunk 3 above)
Comment (hammady) [reply helm/rayyan/values.yaml]: But doesn't 0.1 GB mean 0.1 Gi?

Comment (fabubaker) [reply helm/rayyan/values.yaml]: When they say Gi they mean 0.1GiB (power of 2).

ACI wants it in GB (power of 10).

**Pull request merged**